src/AppStore.ts

```diff
@@ -20,4 +25,13 @@ export class AppStore {
         }
     }
 
+    readonly getPortalVersion = remoteData<string>({
```

Comment:
instead of calling this "getPortalVersion" lets call it just portalVersion ( since it's not a method)